02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,69 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Jake's Webpage</title>
 </head>
 <body>
+  <div align="center">
+    <h1>About Jake!!</h1>
+      <p>Hello!! Jake is the coolest guy around. He somehow manages to work 40 hours at HEB <em><strong>(as a cash controller)</strong></em>, while attending ACA and ACC for a collective 12 more hours per week. Although he is extremely busy, he still finds the time to spend with family and friends.</p>
+  </div>
+  <div>
+    <h1>Jake's Hobbies:</h1>
+      <ul>
```

Comment:
For better readability and error detection put opening and closing tags of block elements on their own line: 
```html
<h1>
   Jake's Hobbies: 
</h1>
<ul>
   ....
</ul>
```
Then you immediately see that the indentation of `ul` is two spaces too much in your case.